app/views/layouts/application.html.haml

```diff
@@ -17,13 +17,12 @@
       = yield :head
 
   %body{'data-environment' => Rails.env}
+    = link_to "Skip to content", "#main", class: "skip-to-content-link"
     #page-top
-    
-
     = render 'layouts/header'
     - if Rails.env.development?
       = render 'layouts/tailwindcss_breakpoints'
-    %main{:class => controller.controller_name}
+    %main{:class => controller.controller_name, id: "main"}
```

Comment:
Having an anchor here would still require tabbing through all the filter items. The goal of LRA-361 is to skip directly to the listing of rooms. If you place the `id="main"` on the div that wraps the `<%= render 'listing' %> in _app/views/rooms/index.html.erb_ view at line 110 you will achieve the proposed solution.

Reply:
Newly proposed solution makes this a moot point